frontend/app/controllers/spree/home_controller.rb

```diff
@@ -3,7 +3,9 @@ class HomeController < Spree::StoreController
     respond_to :html
 
     def index
-      fresh_when etag: store_etag, last_modified: store_last_modified, public: true
+      if Spree::Frontend::Config[:http_cache_enabled]
```

Comment:
@KacperMekarski let's move this to a new helper method `http_cache_enabled?` - let's add this method to `CacheHelper`